README.adoc

```diff
@@ -45,13 +45,12 @@ https://github.com/projectriff/riff-cli[riff-cli] repositories.
 +
 [source, bash]
 ----
-cd $(go env GOPATH)   #defaults to ~/go
-git clone -o upstream https://github.com/projectriff/function-sidecar src/github.com/projectriff/function-sidecar/
-git clone -o upstream https://github.com/projectriff/function-controller src/github.com/projectriff/function-controller/
-git clone -o upstream https://github.com/projectriff/topic-controller src/github.com/projectriff/topic-controller/
-git clone -o upstream https://github.com/projectriff/http-gateway src/github.com/projectriff/http-gateway/
-git clone -o upstream https://github.com/projectriff/shell-function-invoker src/github.com/projectriff/shell-function-invoker/
-git clone -o upstream https://github.com/projectriff/riff-cli src/github.com/projectriff/riff-cli/
+go get github.com/projectriff/function-sidecar
+go get github.com/projectriff/function-controller
+go get github.com/projectriff/topic-controller
+go get github.com/projectriff/http-gateway
+go get github.com/projectriff/shell-function-invoker
+go get github.com/projectriff/riff-cli
```

Comment:
Not a problem with this repo _per se_, but there are two issues with the CLI:
- the installed binary is named `riff-cli`, not `riff`
- this brings a lot of dependencies inside `GOPATH`, I believe because the contents of `vendor/` is not up to date in that repo

Reply:
see https://github.com/projectriff/riff-cli/issues/43